API.md

```diff
@@ -163,3 +163,93 @@ The `indexes` branch keeps a dictionary of indexes to store the status and resul
 Indexing can be skipped by configuring `shouldIndex` to `false` or a function returning `true` or `false` given the index name.  Indexes are initialized in the store lazily, i.e. when they are first seen on an action's `meta.index` property.
 
 Typically `entities` and `indexes` work in tandem to keep track of normalized results from an external data-source while keeping them properly normalized.
+
+## React Bindings
+
+#### `<Provider middleEnd={app} />`
+
+React Context Provider component that handles providing the passed middle-end to any nested Context consumers.
+
+
+It requires the following prop:
+
+- `middleEnd`: A composed application that is set as the context value. The app must be initialized; the component throws otherwise
```

Comment:
Minor nit for consistency with the rest of the docs.
```suggestion
- `middleEnd` - A composed application that is set as the context value. The app must be initialized; the component throws otherwise
```